bedrock/firefox/templates/firefox/features/memory.html

```diff
@@ -0,0 +1,125 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/features/base.html" %}
+
+{# L10n: strings for this page are not considered final and aren't ready for localization. #}
+
+{% block page_title %}{{ _('Less memory usage than Chrome') }}{% endblock %}
+{% block page_title_suffix %}{% endblock %}
+{% block page_desc %}{{ _('If your web browser uses too much memory, switch to Firefox') }}{% endblock %}
+{% block page_image %}{{ static('img/firefox/features/thumbnails/memory.png') }}{% endblock %}
+
+{% block page_css %}
+  {{ super() }}
+  {% stylesheet 'firefox-features-hub-detail' %}
+{% endblock %}
+
+{% block body_id %}firefox-features-fast{% endblock %}
+
+{% block features_header_content %}
+  <h1><span>{{ _('Less memory usage than Chrome') }}</span></h1>
+  <p class="tagline">{{ _('If your web browser uses too much memory, switch to Firefox') }}</p>
+{% endblock %}
+
+{% block features_list %}
+<section class="features-list-section">
+  <div class="content">
+    <h2 class="section-title"><span>{{ _('Features') }}</span></h2>
+
+    <ul class="features-list">
+      <li class="features-list-item">
+        <img src="{{ static('img/firefox/features/thumbnails/memory/fast.png') }}" alt="">
+        <h3>{{ _('Speed up your computer') }}</h3>
+        <p>
+        {% trans %}
+          Every computer program you run takes up some memory. When too much is used,
+          your system can  slooooowww down. Firefox aims for a balance—using enough
```

Comment:
The lowest of nits - an extra space.

Do we care about using `&mdash;` for the dash here?